lib/public/progress-bar.css

```diff
@@ -0,0 +1,52 @@
+.container-progress {
+  width: 600px;
+  margin: 100px auto; 
+}
+.progressbar {
+  counter-reset: step;
+}
+.progressbar li {
+  list-style-type: none;
+  width: 25%;
+  float: left;
+  font-size: 12px;
+  position: relative;
+  text-align: center;
+  text-transform: uppercase;
+  color: #7d7d7d;
```

Comment:
We use this colour a few times throughout this CSS. Looking at MDN, there's a way to set variables in CSS.
This might make it easier to update if the client requests a change in colour during the showcase?
(Even if not, it's interesting that you can set variables in CSS).
https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_variables

Reply:
Thats cool, I will have a look